ddpui/api/metrics_api.py

```
@@ -0,0 +1,443 @@
+@has_permission(["can_edit_charts"])
+def create_or_update_annotation(request, metric_id: int, payload: AnnotationCreate):
+    """Create or update an annotation for a metric + period"""
+    orguser = request.orguser
+    org = orguser.org
+
+    try:
+        metric = MetricDefinition.objects.get(id=metric_id, org=org)
+    except MetricDefinition.DoesNotExist:
+        raise HttpError(404, "Metric not found")
+
+    annotation, created = MetricAnnotation.objects.update_or_create(
```

Ruff (0.15.10)

[warning] Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)